spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/flyway/FlywayMigrationStrategy.java

```diff
@@ -16,7 +16,10 @@
 
 package org.springframework.boot.autoconfigure.flyway;
 
+import org.apache.commons.lang.BooleanUtils;
```

Comment:
We should add a build rule (we did it for guava right, or was that Spring Cloud?).